### ISSUE TITLE
Error Report: Cannot create property 'message' on string 'Node error: {"code":-32000,"message":"transaction underpriced"}'

### ISSUE THREAD

**Issue by Matthewhughes3** [urbit/bridge#413]
## Error

```
TypeError: Cannot create property 'message' on string 'Node error: {"code":-32000,"message":"transaction underpriced"}'
    at renderAdditionalInfo (https://bridge.urbit.org/static/js/main.21612c58.chunk.js:1942:554)
    at ReticketExecute (https://bridge.urbit.org/static/js/main.21612c58.chunk.js:1942:2624)
    at dh (https://bridge.urbit.org/static/js/2.9ed8dd63.chunk.js:159040:7)
    at Jh (https://bridge.urbit.org/static/js/2.9ed8dd63.chunk.js:159516:7)
    at mj (https://bridge.urbit.org/static/js/2.9ed8dd63.chunk.js:161789:86)
    at jj (https://bridge.urbit.org/static/js/2.9ed8dd63.chunk.js:160777:11)
    at Z (https://bridge.urbit.org/static/js/2.9ed8dd63.chunk.js:160634:15)
    at $i (https://bridge.urbit.org/static/js/2.9ed8dd63.chunk.js:160493:16)
    at https://bridge.urbit.org/static/js/2.9ed8dd63.chunk.js:158219:17
    at push.exports.unstable_runWithPriority (https://bridge.urbit.org/static/js/2.9ed8dd63.chunk.js:162914:12)
```

In Bridge version 2.3.5.

**Comment by vvisigoth** [comment]
hey @Matthewhughes3 what were you trying to do when this popped up?

**Comment by liam-fitzgerald** [comment]
@vvisigoth trying to reticket by the looks of the stack trace

@Matthewhughes3 what login method were you using?

@Fang- my gut says this relates to the gas tank and bridge trying to send the same tx

**Comment by liam-fitzgerald** [comment]
should've been fixed with #428
